addons-l10n/en/hovercards.json

```diff
@@ -0,0 +1,28 @@
+{
+  "hovercards/post-count": "{count} posts",
+  "hovercards/loading": "Loading...",
+  "hovercards/error-request-failed": "Request failed. {error}",
+  "hovercards/error-scratchdb": "Error encountered on ScratchDB. {error}",
+  "hovercards/error-no-project": "The project couldn't be found or is private.",
+  "hovercards/error-no-studio": "The studio couldn't be found.",
+  "hovercards/error-no-user": "The user couldn't be found.",
+  "hovercards/error-no-user-scratchdb": "The user couldn't be found in ScratchDB. The user may be too new to be indexed.",
```

Comment:
I think "Cannot find the project" is better than "The project couldn't be found" (personal opinion)